javascript/javascript3/week1/weather/weather.js

```diff
@@ -0,0 +1,99 @@
+const name = document.querySelector("#city");
+const countryName = document.querySelector(".country");
+const temperature = document.querySelector("#temp");
+const icon = document.querySelector("#icon");
+const desc = document.querySelector(".desc");
+const windSpeed = document.querySelector(".wind");
+const humidity = document.querySelector(".humidity");
+const sunRise = document.querySelector(".sunRise");
+const sunSet = document.querySelector(".sunSet");
+const displayCity = document.querySelector(".btn");
+
+const input = document.querySelector("#searchInput");
```

Comment:
CSS classes and ids should be in kebab-case: `searchInput` => `search-input`

Reply:
thank you for telling. I have made the change